[2025-06-29T14:50:23.282+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2025-06-29T14:50:23.373+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-09-10T00:00:00+00:00 [queued]>
[2025-06-29T14:50:23.377+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T14:50:23.548+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-09-10 00:00:00+00:00
[2025-06-29T14:50:23.698+0000] {standard_task_runner.py:57} INFO - Started process 2566 to run task
[2025-06-29T14:50:23.794+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2024-09-10T00:00:00+00:00', '--job-id', '3472', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmph6j096mg']
[2025-06-29T14:50:23.874+0000] {standard_task_runner.py:85} INFO - Job 3472: Subtask insert_into_table
[2025-06-29T14:50:24.439+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2024-09-10T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T14:50:25.156+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-10T00:00:00+00:00'
[2025-06-29T14:50:25.165+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-09-10', 'dag_with_postgres_operator_v03')
        
[2025-06-29T14:50:25.256+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:50:25.337+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T14:50:25.372+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-09-10', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-06-29T14:50:25.375+0000] {sql.py:427} INFO - Rows affected: 1
[2025-06-29T14:50:25.486+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20240910T000000, start_date=20250629T145023, end_date=20250629T145025
[2025-06-29T14:50:25.737+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T14:50:25.836+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
